src/common/function/src/scalars/math/modulo.rs

```diff
@@ -0,0 +1,241 @@
+// Copyright 2023 Greptime Team
```

Comment:
Not sure if that should be updated @waynexia

Reply:
It's all right to not update it.